documental/serializers.py

```diff
@@ -76,42 +77,88 @@ def to_representation(self, instance):
         return url_document
 
 
-class DocumentosTISerializers(serializers.ModelSerializer):
-    """Serializer to return action category `models.DocumentalDocs` data.
+class DocsDocumentTISerializers(serializers.ModelSerializer):
+    """Serializer to return action category `models.DocsDocumentTI` data.
 
-    Data only for the action category linked to DOCUMENTAL_TI.
+    Data only for the action category linked to "DOCUMENTAL_TI".
     """
 
     usercmr_id = UsuarioSerializers()
     action_id = ActionListSerializers()
 
     class Meta:
-        """Meta class for `DocumentosTISerializers` serializer."""
-        model = models.DocumentalDocs
+        """Meta class for `DocsDocumentTISerializers` serializer."""
+        model = models.DocsDocumentTI
         fields = (
             'id_document',
             'path_document',
             'no_document',
-            'no_extension',
             'st_available',
             'st_excluded',
+            'dt_registration',
+            'dt_update',
             'co_funai',
-            'no_ti',
             'co_cr',
             'ds_cr',
+            # 'no_ti',
+            'dt_document',
+            'no_extension',
+            'no_ti',
+            'action_id',
+            'usercmr_id',
+        )
+
+    def to_representation(self, instance):
+        """Method to return in `DocsDocumentTISerializers` the full URL to
+        download the documents in `models.DocsDocumentTI`.
+
+        Returns:
+            str: url to DocumentTI document
+        """
+
+        url_document = super().to_representation(instance)
```

Comment:
Check if this is really necessary.

Reply:
It's necessary. Follow documentation link used to create this code:

https://www.django-rest-framework.org/api-guide/serializers/#advanced-serializer-usage